datadog_checks_base/datadog_checks/base/utils/tracing.py

```diff
@@ -40,7 +40,7 @@ def traced_wrapper(self, *args, **kwargs):
         integration_tracing = is_affirmative(datadog_agent.get_config('integration_tracing'))
 
         if integration_tracing and trace_check:
-            with tracer.trace('integration.check', service='integrations-tracing', resource=self.name):
+            with tracer.trace(self.name, service='integrations-tracing', resource=fn.__name__):
```

Comment:
Maybe add a prefix like `integration.check` before that? In case there are already traces with the same name as an integration.

Reply:
The traces should be pretty uniquely identifiable via other tags like hostname though, so it should still be pretty easy to filter to the desired traces.

Also adding too much to the name can really clutter the flame graph.

Reply:
Right, I was thinking the first argument was the `service`.
All good then 👍